packages/carbon-graphs/src/js/helpers/axis.js

```diff
@@ -514,135 +501,162 @@ const getAxesScale = (axis, scale, config) => {
       return axis;
     }
     // Y and Y2 axes - ticksCount.
+    let yLowerLimit; let yUpperLimit; let y2LowerLimit; let
+      y2UpperLimit;
+    yLowerLimit = config.axis.y.domain.lowerLimit;
+    yUpperLimit = config.axis.y.domain.upperLimit;
+    y2LowerLimit = config.axis.y2.domain.lowerLimit;
+    y2UpperLimit = config.axis.y2.domain.upperLimit;
+
+    // if allowCalibration is true then adjust limits
+    if (config.allowCalibration) {
+      [yLowerLimit, yUpperLimit] = d3
+        .scaleLinear()
+        .domain([yLowerLimit, yUpperLimit])
+        .nice()
+        .domain();
+      [y2LowerLimit, y2UpperLimit] = d3
+        .scaleLinear()
+        .domain([y2LowerLimit, y2UpperLimit])
+        .nice()
+        .domain();
+    }
+    // console.log(yLowerLimit, yUpperLimit, y2LowerLimit, y2UpperLimit);
+
+    let yTickValues; let
+      y2TickValues;
+
     if (
       utils.isDefined(config.ticksCount)
         && config.ticksCount <= constants.TICKSCOUNT_MAXLIMIT
     ) {
-      const yTickValues = generateYAxesTickValues(
-        config.axis.y.domain.lowerLimit,
-        config.axis.y.domain.upperLimit,
+      yTickValues = generateYAxesTickValues(
+        yLowerLimit,
+        yUpperLimit,
         config.ticksCount,
-        config.allowCalibration,
         config.axis.y,
```

Comment:
`generateYAxesTickValues` now takes only three parameters. Does `config.axis.y` still need to be passed?

Reply:
Fixed here: df7d88e6692cb962dbe2020077c85df5b4b27aa6